lista03/ex01/ex01/src/ex01/Veiculo.java

```diff
@@ -0,0 +1,22 @@
+package ex01;
+
+public abstract class Veiculo {
+    
+    protected float preco;
+    protected String modelo;
+    protected String marca;
+    protected String cor;
+    
+    
+    Veiculo (float precoVeiculo, String modeloVeiculo, String marcaVeiculo, String corVeiculo) {
+        preco = precoVeiculo;
+        modelo = modeloVeiculo;
+        marca = marcaVeiculo; 
+        cor = corVeiculo;
+    }
```

Comment:
Acesso de atributos internos faça por meio da keyword this
```suggestion
    Veiculo (float preco, String modelo, String marca, String cor) {
        this.preco = preco;
        this.modelo = modelo;
        this.marca = marca; 
        this.cor = cor;
    }
```